applications/default/public/js/controllers.js

```diff
@@ -304,6 +304,9 @@ function ViewController($scope, $location, $http, applicationState, Choko) {
 
     Choko.get({type: $scope.view.itemType}, function(response) {
       $scope.items = response;
+      $scope.items.$empty = Object.keys($scope.items).filter(function (key) {
```

Comment:
To avoid setting another property on `$scope.items` you can add it to the scope itself like `$scope.empty`.